src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        Ball[] res = new Ball[] {ball1, ball2, ball3};
+
+        for (Ball ball : res) {
+            System.out.println(ball);
+        }
     }
```

Comment:
Remember to use access modifiers for your classes and methods. The 'Application' class and the 'main' method should be declared as 'public'.